CONTRIBUTING.md

```diff
@@ -1,69 +1,42 @@
-
 # Contributing to CloudQuery
 
 :+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
 
-The following is a set of guidelines for contributing to this repository.
-
-## Code of Conduct
-
-This project and everyone participating in it is governed by the [CloudQuery Code of Conduct](https://github.com/cloudquery/cloudquery/blob/main/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. To report inappropriate behavior in violation of the code, please start by reaching out to us on our [Discord channel](https://cloudquery.io/discord).
+This is the repository for CloudQuery SDK. If you are looking for CloudQuery CLI and plugins take a look at our [monorepo](https://github.com/cloudquery/cloudquery)
 
-## I don't want to read this whole thing I just have a question
+## Links
 
-> **Note:** Please don't file an issue to ask a question. You'll get faster results by reaching out to the community on our [Discord channel](https://cloudquery.io/discord)
+- [Homepage](https://cloudquery.io)
+- [Documentation](https://docs.cloudquery.io)
+- [CLI and Plugins Mono Repo](https://github.com/cloudquery/cloudquery)
+- [Discord](https://cloudquery.io/discord)
 
-## What To Know Before Getting Started
+## Development
 
-### CloudQuery Architecture
+### gRPC
```

Comment:
As this is the most "complex" part of the SDK, shouldn't this be last in the guide?